school_evaluation/models/school_evaluation.py

```diff
@@ -164,21 +163,33 @@ class SchoolEvaluationTemplate(models.Model):
     type = fields.Selection([('faculty', 'Faculty'), ('student', 'Student')],
                             'User Type', required=True, default='faculty',
                             help="Type of Evaluation")
-    rating_line = fields.One2many('rating.rating', 'rating_id', 'Rating')
+    # rating_line = fields.One2many('rating.rating', 'rating_id', 'Rating')
+    rating_line_ids = fields.Many2many('rating.rating', 'rating_table')
 
 
 class RatingRating(models.Model):
-    _name = 'rating.rating'
-    _description = "Rating details"
-    _rec_name = 'point'
-    _order = "point desc"
+    _inherit = 'rating.rating'
+
+    @api.model
+    def create(self, vals):
+        res_model = self.env['school.evaluation.template'].browse(
+            vals.get('res_id'))
+        vals.update({'res_model_id': res_model.id})
+        res = super(RatingRating, self).create(vals)
+        return res
+
+    @api.one
+    @api.depends('res_model', 'res_id')
+    def _compute_res_name(self):
+        # cannot change the rec_name of session since it is use to create the bus channel
+        # so, need to override this method to set the same alternative rec_name as rating
+        if self.res_model == 'base':
+            self.res_name = self.rating
+        else:
+            super(RatingRating, self)._compute_res_name()
 
     rating_id = fields.Many2one('school.evaluation.template', 'Stud',
```

Comment:
field name sounds illogical. plz make it meaningful.

Reply:
Improved now.